### DISCUSSION TITLE
Sending Feedback to Sentry, how can I separate Feedback submissions into Issues, not Events?

### DISCUSSION THREAD

**Discussion by TaylorHo** [ueman/feedback#237] [Q&A] [answered]
By default, when sending two or more Feedbacks to Sentry, they are treated as Events, grouped into a unique Issue.

I want to change this behavior so that each Feedback sent is treated as a unique Issue, **not grouped events**.

Do you know how I can do this?

**Comment by ueman** [comment, marked as answer]
I hope this will be possible as soon as Sentry tackles https://github.com/getsentry/sentry/discussions/54956

I'm afraid right now it's not really possible.
